.githooks/pre-commit

```diff
@@ -101,4 +101,18 @@ then
   done
 fi
 
+# Check Apache License header for source files.
+if [ $NUM_JAVA_FILES_CHANGED -gt 0 ]
+then
+  echo_status "Checking Apache License Header ..."
+  go get -u github.com/google/addlicense
```

Comment:
Could we add some extra logic to do the following? We can pull this out into a separate method.

- Check whether the current system has a go path, e.g. ~/go/bin
- If not, print a link to set-up or installation steps and exit
- Otherwise, check if the go path is in the system path.
  - If not, add the go path to the system path.
- Check if addlicense exists and install addlicense only if it's not there.

Reply:
sure, working on it.

Reply:
done.